CHANGELOG

```diff
@@ -1,6 +1,10 @@
+Version 0.3.2 (2024-08-21)
+--------------------------
+Credit Original Author (#11)
```

Comment:
@jbeemster added a new suggestion here, and deleted the other non-changing commit suggestion.
```suggestion
Credit Original Author, Teghan Nightengale (#11)
```